src/DataLoaders.py

```diff
@@ -0,0 +1,102 @@
+# Classes for wrapping data into datasets
+
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from tqdm import tqdm
+from typing import Optional, Literal
+from torch_geometric.data import Dataset
+
+class ProteinDataset(Dataset):
+    """
+    Dataset for loading proteins from the paper given splits.
+    """
+    def __init__(self, labels, database, include, flex: Literal["msqf", "bfact", "pseudo"] = "msqf", exclude: Optional[str] = None):
```

Comment:
Please add a docstring.
Please add type-hinting to the remaining arugments in the signature.